src/common/commandExecutor.ts

```diff
@@ -10,13 +10,20 @@ import {HostPlatform, HostPlatformId} from "../common/hostPlatform";
 import {ErrorHelper} from "./error/errorHelper";
 import {InternalErrorCode} from "./error/internalErrorCode";
 
+export enum CommandVerbosity {
+    OUTPUT,
+    SILENT,
+    PROGRESS,
+}
+
 interface EnvironmentOptions {
     REACT_DEBUGGER?: string;
 }
 
 interface Options {
     env?: EnvironmentOptions;
-    silent?: boolean;
```

Comment:
Did a search in repo and looks like this silent is never used. Makes sense to remove it.